packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts

```diff
@@ -60,6 +66,21 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
       throw new NotFoundException('Object does not exist');
     }
 
+    // Double check in case the service is directly called
+    if (isEnumFieldMetadataType(fieldMetadataInput.type)) {
```

Comment:
I feel there is something wrong here: isEnumFieldMetadataType does not make sense to me
I propose areEnumOptionsRequired() and Rating field should not be part of it.

It's weird to include Rating to then exclude it

Reply:
@charlesBochet I can create another util for that, it's just that `isEnumFieldMetadataType` already exist and is used in other places